### ISSUE TITLE
Include validation set example

### ISSUE THREAD

**Issue by danielkelshaw** [danielkelshaw/torchNetworks#11]
At the moment the current implementations are using the testset for validation - it would be worth including an example where the training set is split to include validation data as well.
